projects/addon-preview/components/preview/action/preview-action.style.less

```diff
@@ -0,0 +1,25 @@
+@import 'taiga-ui-local';
+@import '../mixin.style.less';
+
+:host {
+    display: inline-block;
+    margin-left: 0.3125rem;
+    margin-right: 0.3125rem;
+}
+
+.t-button {
+    .preview-item();
+    .transition(background);
+
+    &:hover {
+        background: rgba(159, 159, 159, 0.86);
```

Comment:
do we have similar color from our palette ?

Reply:
see [comment](https://github.com/TinkoffCreditSystems/taiga-ui/pull/350#discussion_r741807150)